engines/advancedDetector.cpp

```diff
@@ -576,6 +639,85 @@ static bool getFilePropertiesIntern(uint md5Bytes, const AdvancedMetaEngine::Fil
 	return true;
 }
 
+struct ParsedFileDesc {
+	const char *fileName;
+	const char *md5;
+	int64 fileSize;
+
+	bool is_tailmd5;
```

Comment:
I do not undestand, why use these flags instead of a bitfield?

Reply:
You mean like adding:1 after names? I can do this

Reply:
No, put everything into `uint32` and then use ADGF_TAILMD5 etc.